### PR TITLE
Separated CBA and ACE3 macros and streamlined

### LINKED ISSUE #19: Verify which macros are ACE3's and which CBA's
Currently it's **not** correct.

### PR DESCRIPTION
**Pull request by jonpas**
Closes #19

### CODE REVIEW

File: settings/language-sqf-macros-cba.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-1-module-pbo-specific-macro-usage"
+            }, {
+              "description": "Expands to: prefix_component_variable",
+              "snippet": "EGVAR(${1:component},${2:variable})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-1-module-pbo-specific-macro-usage"
+            }, {
+              "description": "Expands to: prefix_component_fnc_functionName",
+              "snippet": "FUNC(${1:functionName})$0",
```
Comment (jokoho48) [inline settings/language-sqf-macros-cba.json:40]: this macro is edit by ace3 and cant use as befor same with EFUNC

File: settings/language-sqf-macros-cba.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+              "snippet": "ELSTRING(${1:component},${2:string})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-2-2-string-family-macros"
+            }, {
+              "description": "Expands to: \"$STR_prefix_component_fnc_string\"",
+              "snippet": "CSTRING(${1:string})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-2-2-string-family-macros"
+            }, {
+              "description": "Expands to: \"$STR_prefix_component_fnc_string\"",
```
Comment (jokoho48) [inline settings/language-sqf-macros-cba.json]: fnc? is this not `$STR_prefix_component_string`

File: settings/language-sqf-macros-cba.json (hunk 1 above)
Comment (jonpas) [inline settings/language-sqf-macros-cba.json:40]: What's the difference? I know ACE3 adds debug level, but that's not something I'd add to this auto-completion. Example would be nice.

File: settings/language-sqf-macros-cba.json (hunk 2 above)
Comment (jonpas) [inline settings/language-sqf-macros-cba.json]: Must be a bad copy from conversion to autocomplete-plus, thanks.

File: settings/language-sqf-macros-cba.json (hunk 1 above)
Comment (jokoho48) [inline settings/language-sqf-macros-cba.json:40]: i mean only that we proberly move this to ace macros or add a comment that if you use ace this act different

Comment (jonpas) [inline settings/language-sqf-macros-cba.json:40]: @jokoho48 You can create a new issue if you see this is a problem or we should add something.

**Pull request merged**